src/ui/components/Shared/TracksList.react.js

```diff
@@ -329,20 +330,12 @@ export default class TracksList extends Component {
   }
 
   render() {
-    const tracks = [...this.props.tracks];
+    const { tracks, type } = this.props;
 
-    // TODO (y.solovyov | KeitIG): TrackListHeader component?
     return (
-      <div className='tracks-list-container'> {/* there used to be a tabIndex={ 0 }, I don't remember why */}
+      <div className='tracks-list'> {/* there used to be a tabIndex={ 0 }, I don't remember why */}
         <KeyBinding onKey={this.onKey} preventInputConflict />
-        <div className='tracks-list-header'>
-          <div className='track-cell-header cell-track-playing' />
-          <div className='track-cell-header cell-track'>Track</div>
-          <div className='track-cell-header cell-duration'>Duration</div>
-          <div className='track-cell-header cell-artist'>Artist</div>
-          <div className='track-cell-header cell-album'>Album</div>
-          <div className='track-cell-header cell-genre'>Genre</div>
-        </div>
+        <TracksListHeader enableSort={type === 'library'} />
```

Comment:
Sorting is not supported in playlists?

Reply:
So yeah, that is one of the problem I faced. The thing with playlist is do not want them to be sortable, I want them to be re-orderable (#175).

Let's imagine we have sortable and re-orderable playlist, switching from one mode to the other will introduce a lot of complexity. I checked other players, and most of them do not allow you to sort playlists.

Reply:
Fair.